### ISSUE TITLE
Does this project have a version number?

### ISSUE THREAD

**Issue by yurivict** [m-a-d-n-e-s-s/madness#340]
```git describe``` shows: ```last_svn-2744-g35077935```

Could you please consider making a release with a conventional version number format?

**Comment by robertjharrison** [comment]
Yeah ... this has already been identified as an issue but we've not yet formalized the release process ... we anticipate doing so this summer.  The project really consists of 3 things that need pulling apart (the parallel runtime, the multiresolution numerical library, and the science applications built on the numerical library).

Which piece are you most interested in being versioned?

**Comment by yurivict** [comment]
I am most interested in a multiresolution numerical library, but once you will split it all of them should be versioned.

**Comment by mbanck** [comment]
Any update on this?
